server/sonar-web/src/main/js/apps/background-tasks/app.js

```diff
@@ -19,22 +19,26 @@
  */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
+import { Router, Route, Redirect, useRouterHistory } from 'react-router';
+import { createHistory } from 'history';
 
-import BackgroundTasksAppContainer from './containers/BackgroundTasksAppContainer';
-import rootReducer from './store/reducers';
-import configureStore from '../../components/store/configureStore';
-
-import './styles/background-tasks.css';
+import BackgroundTasksApp from './components/BackgroundTasksApp';
 
 window.sonarqube.appStarted.then(options => {
   const el = document.querySelector(options.el);
 
-  const store = configureStore(rootReducer);
+  const history = useRouterHistory(createHistory)({
+    basename: window.baseUrl + (options.component ? '/project/background_tasks' : '/background_tasks')
+  });
+
+  const App = (props) => {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Remove parentheses around the parameter of this arrow function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)